tests/test_correct_year.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+"""
+    test_correct_year
```

Comment:
This should be moved into `test_build_gettext.py`.
It is testcase for feature of gettext builder. It might be separated to other package or subdirectories.